doc/from.md

```diff
@@ -28,8 +28,8 @@ const outer = new SimpleMap({
 });
 
 const keyObservable = canValue.from(outer, "inner.key");
-// canReflect.getValue(keyObservable) === "hello"
+// keyObservable.value === "hello"
 
-canReflect.setValue(keyObservable, "aloha");
+keyObservable.value = "aloha";
```

Comment:
Here’s what this section looks like:
<img width="755" alt="screen shot 2018-07-02 at 7 30 01 pm" src="https://user-images.githubusercontent.com/10070176/42195610-8e1f2762-7e2e-11e8-86bd-2d7a8ff86e4a.png">